mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/MailboxMapper.java

```diff
@@ -99,4 +100,10 @@ boolean hasChildren(Mailbox mailbox, char delimiter)
      * @throws MailboxException 
      */
     List<Mailbox> list() throws MailboxException;
+
+    List<MailboxAnnotation> getAnnotations(Mailbox mailbox) throws MailboxException;
```

Comment:
Sometime I confuse, for the first version, I add javadoc but Benoit said we don't need it.
So do we need javadoc for the public method?